rmf_site_editor/src/workspace.rs

```diff
@@ -282,6 +284,35 @@ pub fn dispatch_load_workspace_events(
                     })
                     .detach();
             }
+            LoadWorkspace::BlankFromDialog => {
+                let sender = load_channels.sender.clone();
+                #[cfg(not(target_arch = "wasm32"))]
+                {
+                    AsyncComputeTaskPool::get()
+                        .spawn(async move {
+                            if let Some(file) = AsyncFileDialog::new().save_file().await {
+                                let file = file.path().to_path_buf();
+                                let name = file
+                                    .file_stem()
+                                    .map(|s| s.to_str().map(|s| s.to_owned()))
+                                    .flatten()
+                                    .unwrap_or_else(|| "blank".to_owned());
+                                let data = WorkspaceData::LoadSite(LoadSite::blank_L1(
+                                    name,
+                                    Some(file.clone()),
+                                ));
```

Comment:
If this is the preferred default I wonder if we should also adopt it in the function that gets called when users click `File->New` or press `Ctrl-N`, since right now the behavior would be different (so basically having the same [here](https://github.com/open-rmf/rmf_site/blob/269efabaa7def79aa3d827606e2cd21d67960dc3/rmf_site_editor/src/workspace.rs#L237-L248))

Reply:
Good point, I've standardized that now by pushing the creation of a blank site upstream: [9e66538](https://github.com/open-rmf/rmf_site/pull/220/commits/9e6653876a49fa9a780597a210b64076f2e0425a)